openwisp_controller/config/base/config.py

```diff
@@ -71,6 +71,12 @@ class AbstractConfig(BaseConfig):
             '"error" means the configuration caused issues and it was rolled back;'
         ),
     )
+    error_reason = models.CharField(
+        _('error reason'),
+        max_length=1050,
```

Comment:
Let's use `1024`, please also make sure that in case it's longer than that it gets automatically truncated without raising validation error so that we avoid incurring the case in which the status doesn't get reported because for an unexpected reason the error log sent is too long (it shouldn't happen because we'll be truncating it in openwisp-config too, but as I stress, a little bit of redundancy in these matters of robustness is always good).